wildfly-subsystem-archetype/src/main/resources/archetype-resources/__artifactId__-feature-pack/src/main/resources/modules/system/layers/base/main/module.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
modules/system/layers/base is for WildFly core modules, that is perhaps just fine. But you could add your module at the root of the modules directory.
In such a case, modules.all would be not available, you would use the module name as default package.